specs/driver_spec.rb

```diff
@@ -75,6 +75,111 @@
     it "returns zero if no trips" do
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
       driver.average_rating.must_equal 0
+    end # ends 'it "returns zero if no trips" do'
+  end # ends 'describe "average_rating method" do'
+
+
+  describe "total_revenue method" do
+    it "calculates the total revenue" do
+      trips = [
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?